aten/src/ATen/functorch/BatchedTensorImpl.cpp

```diff
@@ -128,9 +128,19 @@ SymIntArrayRef BatchedTensorImpl::sym_strides_custom() const {
 // TODO: implement proper contiguity on batched tensor, then put
 // sizes_strides_policy back to Default
 bool BatchedTensorImpl::is_contiguous_custom(at::MemoryFormat memory_format) const {
-  TORCH_CHECK(memory_format == MemoryFormat::Contiguous,
-      "NYI: querying is_contiguous inside of vmap for memory_format ",
-      "other than torch.contiguous_format");
+  // TORCH_CHECK is disable for now as there is some code in
+  // torch/_dynamo/tensor.py::TensorVariable::specialize that calls `is_contiguous_custom`
+  // with different memory formats. Since the only memory format supported on
+  // BatchedTensors is MemoryFormat::Contiguous, PyTorch just return false for
+  // the other ones.
+
```

Comment:
Can you link the code? This sounds suspicious

Reply:
Sure, here is the stacktrace if the TORCH_CHECK is enabled:
https://gist.github.com/guilhermeleobas/3a45544e201461a12eb9712ac889d1cb

Reply:
Okay, we shouldn't be changing BatchedTensor's is_contiguous semantics. Instead, somehow we want to pass is_contiguous=None to the TensorVariable when it gets built

Reply:
Is this changing the `is_contiguous` semantics? Rather than raising an exception, this change returns `False` for any case that it is not `torch.contiguous_format`.

Reply:
That is changing the semantics, because instead of an error we end up doing something, right?